release-notes/1.19.1.md

```diff
@@ -0,0 +1,38 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.19.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+* The `anbox-cloud-appliance status` command now lists the version of the application that you are running.
+* The following snaps will be held from being automatically refreshed. They are refreshed only when the charms are upgraded.
+    - `ams`
+    - `ams-node-controller`
+    - `anbox-cloud-dashboard`
+    - `anbox-stream-agent`
+    - `anbox-stream-gateway`
+    - `lxd`
+* You can now see when an application or its version was created using the `CreatedAt` field listed when you run `amc application show`.
+* Android security updates for September 2023 (see Android Security Bulletin - September 2023 for more information).
```

Comment:
Android security updates for September 2023 (see Android Security Bulletin - September 2023 for more information).
->
Android security updates for September 2023 (see [Android Security Bulletin - September 2023](https://source.android.com/docs/security/bulletin/2023-09-01) for more information).